source/common/config/remote_data_fetcher.cc

```diff
@@ -44,15 +44,15 @@ void RemoteDataFetcher::onSuccess(const Http::AsyncClient::Request&,
   const uint64_t status_code = Http::Utility::getResponseStatus(response->headers());
   if (status_code == enumToInt(Http::Code::OK)) {
     ENVOY_LOG(debug, "fetch remote data [uri = {}]: success", uri_.uri());
-    if (response->body()) {
+    if (response->body().length() > 0) {
```

Comment:
Might be worth it to add an `empty()` method to `Buffer` at some point.

Reply:
haha yeah, I'm so used to getting reminded to use empty() over length() > 0 I started with that and was surprised when it didn't compile :-P